### DISCUSSION TITLE
How to save everything to one file

### DISCUSSION THREAD

**Discussion by thibaultmol** [Serene-Arc/bulk-downloader-for-reddit#798] [Q&A] [answered]
I'm trying to download all my saved posts. I just want one file that contains all the link so the saved posts. Can't seem to figure out how to do that though

**Comment by Serene-Arc** [comment, marked as answer]
That's not really something that the BDFR does. I think you'd be best served filing a GPDR data request with Reddit; that will give you all of the IDs that you've saved.
